frontend/src/components/LineChart/LineChart.js

```diff
@@ -36,6 +44,7 @@ const LineChart = ({ data, title, xAxis, yAxis }) => {
     <Box ref={ref}>
       <VictoryChart
         {...size}
+        scale="time"
```

Comment:
It looks like this should be `scale={{x: "time", y: "linear"}}`. Setting both axes to `"time"` does something odd to the dashed lines of the y-axis.

Also, this should only be `"time"` when the variant is `time` or `time_years`.

Same for `<StackChart>`.